modules/dicom_archive/php/dicom_archive.class.inc

```diff
@@ -48,6 +48,9 @@ class Dicom_Archive extends \NDB_Menu_Filter
      */
     function _setupVariables()
     {
+        // Rest the filter because this is broken with react modules
+        $this->_resetFilters();
```

Comment:
The issue is, is that filters are done in the browser so if you reload the page in the current implementation you lose your filter anyways.This is just a simple fix so that you dont only get part of the data when you're coming from another page where a filter is set. The next itteration will save the filter in the session variable in the backend and load it into the react component to filter properly